README.rst

```diff
@@ -21,6 +21,8 @@ Please use the `main ZMON repository`_ to start a Vagrant demo box.
 
 Make sure the provided Vagrant-Box is up and all services are running.
 
+Update your GitHub ``client-id`` and ``client-secret`` in ``zmon-controller-app/src/main/config/application-github.yml``.
```

Comment:
The shipped configuration contains `YourGitHubClientID` and `YourGitHubClientSecret`. You usually find it out only after running the application once. I guess there's also an easier way via env variables?